lib/collections/schemas/products.js

```diff
@@ -270,22 +271,24 @@ export const ProductVariant = new SimpleSchema({
     optional: true
   },
   taxDescription: {
-    type: String,
+    type: Intlfield,
```

Comment:
This is more of an internal field, do we actually need this to be translated if customers never see it? Was your thinking to just change everything that's a `String` to `Intlfield`?